test_runner/src/main/kotlin/ftl/args/ArgsHelper.kt

```diff
@@ -42,20 +42,18 @@ object ArgsHelper {
     }
 
     fun assertFileExists(file: String, name: String) {
-        // flank can't detect (for now) if files exists on GCS, therefore we want to skip validation
-        // todo https://github.com/Flank/flank/issues/1099
-        if (file.startsWith(GCS_PREFIX)) return
-
-        if (!File(file).exists()) {
+        if (!file.exist())
```

Comment:
Again, totaly cosmetic change, IMO can improve readability a bit. How about `exists().not()` or `notExists()` ? I mean, this negation mark can be skipped by mistake. It's not a change request :)